### PR TITLE
Add blog post draft for the 0.3 release announcement.

### PR DESCRIPTION
**Pull request by mirkoboehm**
Comments welcome.

### CODE REVIEW

**Review by GiasemiSh — APPROVED** [review]
lgtm

File: content/blog/qmstr-milestone-3-report.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+banner = "img/blog/qmstr-ms-3-report-title.jpg"
++++
+
+Version 0.3 of Quartermaster was tagged on February 1, 2019. It
+includes improvements to allow for example the Linux kernel, glibc and
+openssl as projects under analysis. It delivers support for snapshots
+of the knowledge graph, which allows to roll back changes to a known
+state, as well as support for source code in assembly language. It
+improves the support for `ccache`, `ar`, `ld` and `objcopy`, and for
+analysing source code elements that are generated during the build and
```
Comment (thegrumpylion) [inline content/blog/qmstr-milestone-3-report.md]: analysing -> analyzing

File: content/blog/qmstr-milestone-3-report.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+date = "2019-02-03T00:00:00+01:00"
+tags = ["qmstr", "FOSS", "compliance", "milestone", "report"]
+categories = ["qmstr"]
+banner = "img/blog/qmstr-ms-3-report-title.jpg"
++++
+
+Version 0.3 of Quartermaster was tagged on February 1, 2019. It
+includes improvements to allow for example the Linux kernel, glibc and
+openssl as projects under analysis. It delivers support for snapshots
+of the knowledge graph, which allows to roll back changes to a known
```
Comment (thegrumpylion) [inline content/blog/qmstr-milestone-3-report.md]: to roll -> rolling

File: content/blog/qmstr-milestone-3-report.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+
+Larger projects also take longer to compile, so we made sure to
+improve the integration with ccache as a means to speed up
+compilation. Scancode integration has been extended to integrate the
+author information it produces into the knowledge graph. Work
+continues on streamlining the graph model. Nodes are now typed using a
+feature of the underlying DGraph database. This effort will continue
+throughout the next release, aiming for more streamlining of the data
+model that allows for more generic approaches to reporting aggregated
+results. Support for the C++ programing language was extended through
```
Comment (thegrumpylion) [inline content/blog/qmstr-milestone-3-report.md]: programing -> programming

File: content/blog/qmstr-milestone-3-report.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+Commission H2020](https://ec.europa.eu/programmes/horizon2020/) funded
+consortium that develops innovative ecosystem analysis techniques for
+C/C++, Java and Python applications. [Endocode](https://endocode.com/)
+is a member of the consortium and will work on further development of
+Quartermaster as part of the project for the coming three years. Work
+on the project began in January 2019 with a kick-off meeting at [TU
+Delft](https://www.tudelft.nl/en/).
+
+Endocode has aimed at developing Quartermaster under a neutral
+umbrella organisation form the start. With the upcoming launch of
```
Comment (thegrumpylion) [inline content/blog/qmstr-milestone-3-report.md]: form -> from

**Review by thegrumpylion — CHANGES_REQUESTED** [review]

File: content/blog/qmstr-milestone-3-report.md [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+development team focused on covering complete use cases of real-life
+Open Source projects like the Linux kernel, openssl or glibc. This
+approach helped to identify gaps in Quartermaster functionality which
+led, for example, to added functionality for `ld` and `objcopy` that
+are used in the build system of these projects. One concern in such
+scenarios is that code gets either generated or combined from other
+elements in the source tree and then passed to the compiler via stdin,
+without being saved to the file system. In such a case it is difficult
+to identify what exact code is compiled and added to the resulting
+binary. The code is now saved in the knowledge graph for inspection.
```
Comment (hemarkus) [inline content/blog/qmstr-milestone-3-report.md]: It is actually not saved in the graph but in the file system to get checked by e.g. scancode

File: content/blog/qmstr-milestone-3-report.md [hunk 6]
```diff
@@ -0,0 +1,150 @@
...
+
+Larger projects also take longer to compile, so we made sure to
+improve the integration with ccache as a means to speed up
+compilation. Scancode integration has been extended to integrate the
+author information it produces into the knowledge graph. Work
+continues on streamlining the graph model. Nodes are now typed using a
+feature of the underlying DGraph database. This effort will continue
```
Comment (hemarkus) [inline content/blog/qmstr-milestone-3-report.md]: it is not a feature of dgraph but more a best practice to model your structs

**Review by hemarkus — CHANGES_REQUESTED** [review]
minor changes required

File: content/blog/qmstr-milestone-3-report.md (hunk 4 above)
Comment (Katajau) [inline content/blog/qmstr-milestone-3-report.md]: umbrella organisation from (or is form correct?) the start.

**Review by Katajau — COMMENTED** [review]
Looks great, just one small typo.

File: content/blog/qmstr-milestone-3-report.md (hunk 4 above)
Comment (mirkoboehm) [reply content/blog/qmstr-milestone-3-report.md]: Fixed.

**Pull request merged**